src/utils/openmc/statepoint.py

```diff
@@ -7,6 +8,9 @@
 import openmc
 from openmc.constants import *
 
+if sys.version > '3':
```

Comment:
I'm inclined to leave Travis using 2.7 for the time being. Most (all except Arch?) distributions use 2 as default still. @scopatz do you have any thoughts about good ways to test both 2 and 3?